### PR TITLE
Add iCamSource.app v2.8.5

### PR DESCRIPTION
**Pull request by Saklad5**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit —download {{cask_file}}` is error-free.
- [x] `brew cask style —fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/icamsource.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    url "http://skjm.com/icam/iCamSource#{version}.dmg"
+    sha256 'ff27ea1747f6ea555b4f98e01a2a57c6a2c939bd06cdf28211b423e6c4af2704'
+  end
+  appcast 'http://skjm.com/sparkle/iCamSource.xml',
+          checkpoint: '67a506f70a3a997b2d77198ca8902b5fdf7bdfd7a28c269ffebcc8528bb58650'
+  name 'iCamSource'
+  homepage 'http://skjm.com/icam/support.php'
+  license :gratis
+
+  depends_on macos: '>= :tiger'
```
Comment (vitorgalvao) [inline Casks/icamsource.rb]: You can remove this. HBC doesn’t even work that far back.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/icamsource.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+cask 'icamsource' do
+  if MacOS.version <= :mountain_lion
+    version '2.7.2'
+    if MacOS.version <= :leopard
```
Comment (vitorgalvao) [inline Casks/icamsource.rb]: You can remove this conditional as well. Anything below Snow Leopard isn’t worth worrying about.

File: Casks/icamsource.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    sha256 'ff27ea1747f6ea555b4f98e01a2a57c6a2c939bd06cdf28211b423e6c4af2704'
+  end
+  url "http://skjm.com/icam/iCamSource#{version}.dmg"
+  appcast 'http://skjm.com/sparkle/iCamSource.xml',
+          checkpoint: '67a506f70a3a997b2d77198ca8902b5fdf7bdfd7a28c269ffebcc8528bb58650'
+  name 'iCamSource'
+  homepage 'http://skjm.com/icam/support.php'
+  license :gratis
+
+  depends_on macos: '>= :snow_leopard'
```
Comment (vitorgalvao) [inline Casks/icamsource.rb]: What I meant is that you can delete this line entirely.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/icamsource.rb (hunk 3 above)
Comment (Saklad5) [inline Casks/icamsource.rb]: The most recent version of iCamSource supported by Mountain Lion is not supported by anything earlier than Snow Leopard.

Comment (vitorgalvao) [inline Casks/icamsource.rb]: But neither is Homebrew-Cask. Even if your system is pre-Snow Leopard, you won’t be able to use this cask anyway, meaning the line does not need to exist.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/icamsource.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    version '2.8.5'
+    sha256 'ff27ea1747f6ea555b4f98e01a2a57c6a2c939bd06cdf28211b423e6c4af2704'
+  end
+  url "http://skjm.com/icam/iCamSource#{version}.dmg"
+  appcast 'http://skjm.com/sparkle/iCamSource.xml',
+          checkpoint: '67a506f70a3a997b2d77198ca8902b5fdf7bdfd7a28c269ffebcc8528bb58650'
+  name 'iCamSource'
+  homepage 'http://skjm.com/icam/support.php'
+  license :gratis
+
```
Comment (vitorgalvao) [inline Casks/icamsource.rb]: There’s an extraneous empty line you need to remove.

**Comment by Saklad5** [comment]
I wish Travis Cl would discard commits that have already been superseded before they were started.

**Review by vitorgalvao — APPROVED** [review]
Can be merged as is if Travis passes, if there are no changes after this approval.

**Comment by vitorgalvao** [comment]
Thank you for the submission and the fast changes.

**Pull request merged**